assets/js/util/tracking/index.js

```diff
@@ -30,13 +30,23 @@ const {
 	userIDHash,
 } = global._googlesitekitBaseData || {};
 
+const {
+	user: { roles: currentUserRoles },
+	authenticated: authenticatedUser,
+} = global._googlesitekitUserData || {};
+
+const { GOOGLESITEKIT_VERSION: pluginVersion } = global;
+
 const initialConfig = {
 	activeModules,
 	trackingEnabled,
 	trackingID,
 	referenceSiteURL,
 	userIDHash,
 	isSiteKitScreen,
+	currentUserRoles,
```

Comment:
We can leave off the `current` as everything related to the user here is already specific to the current user (e.g. `userIDHash`)
```suggestion
	userRoles,
```

Reply:
👍